src/AppInstallerCommonCore/Fonts.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include <dwrite_3.h>
+#include <AppInstallerStrings.h>
+#include <winget/Fonts.h>
+
+namespace AppInstaller::Fonts
+{
+    namespace
+    {
+        std::vector<std::filesystem::path> GetFontFilePaths(const wil::com_ptr<IDWriteFontFace>& fontFace)
```

Comment:
Seeing all upper case for returned file paths, can we update the casing for readability?

```
~ wingetdev font list --family-name "CaskaydiaCove Nerd Font Mono"
Face               Family                       Paths
-----------------------------------------------------------------------------------------------------------------------------------------------------------
book               CaskaydiaCove Nerd Font Mono C:\USERS\<USER>\APPDATA\LOCAL\MICROSOFT\WINDOWS\FONTS\CASKAYDIA COVE NERD FONT COMPLETE MONO.TTF
regular            CaskaydiaCove Nerd Font Mono C:\USERS\<USER>\APPDATA\LOCAL\MICROSOFT\WINDOWS\FONTS\CASKAYDIA COVE NERD FONT COMPLETE MONO REGULAR.OTF
italic             CaskaydiaCove Nerd Font Mono C:\USERS\<USER>\APPDATA\LOCAL\MICROSOFT\WINDOWS\FONTS\CASKAYDIA COVE NERD FONT COMPLETE MONO ITALIC.OTF
```

Reply:
May also need to consider anonymizing path...

Reply:
If we are given upper case from the API, I'm not sure it is worth the effort to use the filesystem to correct it back to the original casing.

Reply:
> May also need to consider anonymizing path...

And respecting the AnonymizeDisplayPaths setting